e2e/utils/validators.ts

```diff
@@ -5,6 +5,7 @@ import fetch from 'node-fetch';
 type ValidationOptions = {
   isIndexedRAMBundle?: boolean;
   platform?: string;
+  log?: boolean;
```

Comment:
Is this option used anywhere?

Reply:
No, I forgot about it when I was removing whole log option implementation